RadixWallet/Features/InteractionReview/Sections/Accounts/Accounts.swift

```diff
@@ -68,6 +68,7 @@ extension InteractionReview {
 			let account: InteractionReview.ReviewAccount
 			var transfers: IdentifiedArrayOf<InteractionReview.Transfer>
 			let isDeposit: Bool
+			var isAccountDelete: Bool = false
```

Comment:
`isAccountDelete` sounds wrong, perhaps `isDeletingAccount` or `forDeletionOfAccount`?

Reply:
wait can `isDeposit` and `isAccountDeletion` be **both** `true`? else I suggest you introduce an enum a `Purpose`?

Reply:
Right, will add an enum